models/sessions/scratch/snowflake/snowplow_web_sessions_this_run.sql

```diff
@@ -92,6 +96,12 @@ with session_firsts as (
         -- event name for use later
         event_name
     from {{ ref('snowplow_web_base_events_this_run') }}
+    left join
+        {{ ref('ga4_source_categories') }} c on mkt_source = c.source
+    left join
+        {{ ref('rfc_5646_language_mapping') }} l on br_lang = tag
+    left join
+        {{ ref('geo_country_mapping') }} g on ev.geo_country = g.alpha_2
```

Comment:
These joins should be done on `lower/upper` versions of the text where sensible, it may just be our integration tests data but that has different cases on br-lang which may be possible to get in the wild? Same on bigquery
<img width="1049" alt="image" src="https://github.com/snowplow/dbt-snowplow-web/assets/8260415/c842c80e-7317-47fa-b9ee-726649f58982">